src/app/index.config.coffee

```diff
@@ -8,7 +8,7 @@ angular.module 'mnoEnterpriseAngular'
     # Set options third-party lib
     toastrConfig.timeOut = 5000
     toastrConfig.positionClass = 'toast-top-right'
-    toastrConfig.preventDuplicates = true
+    toastrConfig.preventDuplicates = false
```

Comment:
Noox is there a reason this is set to true in the framework? it prevents alerting non-unique messages.